query-engine/connector-test-kit/src/test/scala/writes/topLevelMutations/DeleteMutationRelationsSpec.scala

```diff
@@ -7,103 +7,6 @@ import util._
 class DeleteMutationRelationsSpec extends FlatSpec with Matchers with ApiSpecBase with SchemaBaseV11 {
   override def runOnlyForCapabilities: Set[ConnectorCapability] = Set(JoinRelationLinksCapability)
 
-  "a P1! to C1! relation " should "error when deleting the parent" in {
-    schemaWithRelation(onParent = ChildReq, onChild = ParentReq).test { t =>
-      val project = SchemaDsl.fromStringV11() {
-        t.datamodel
-      }
-      database.setup(project)
-
-      server
-        .query(
-          s"""mutation {
-          |  createChild(data: {
-          |    c: "c1"
-          |    c_1: "c_1"
-          |    c_2: "c_2"
-          |    parentReq: {
-          |      create: {
-          |        p: "p1"
-          |        p_1: "p_1"
-          |        p_2: "p_2"
-          |      }
-          |    }
-          |  }){
-          |    ${t.child.selection}
-          |  }
-          |}""",
-          project
-        )
-
-      server.queryThatMustFail(
-        s"""
-         |mutation {
-         |  deleteParent(
-         |    where: {p: "p1"}
-         |  ){
-         |    p
-         |  }
-         |}
-      """,
-        project,
-        errorCode = 2014,
-        errorContains = """The change you are trying to make would violate the required relation 'ChildToParent' between the `Child` and `Parent` models"""
-      )
-
-    }
-  }
-
-  "a P1! to C1! relation " should "error when deleting the parent2" in {
```

Comment:
This looks like a dedicated test case for required 1:1 relations that we don't need anymore.

Reply:
Jep, that can go.